taqueria-plugin-flextesa/README.md

```diff
@@ -59,7 +60,7 @@ This example shows the configuration for the default sandbox named `local`:
     "sandbox": {
         "local": {
             "label": "Local Tezos Sandbox",
-            "protocol": "PtHangz2aRngywmSRGGvrcTyMbbdpWdpFKuS4uMWxg2RaH9i1qx",
+            "protocol": "Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A",
```

Comment:
Should this be updated to Jakarta?

Reply:
Yes, our config defaults to ithaca still afaik so I chose that for consistency but I think we should update all protocol references to Jakarta